core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -35,14 +38,27 @@ object Validator extends ValidatorMacros {
 
   def min[T: Numeric](value: T, exclusive: Boolean = false): Validator.Primitive[T] = Min(value, exclusive)
   def max[T: Numeric](value: T, exclusive: Boolean = false): Validator.Primitive[T] = Max(value, exclusive)
+  def zero[T: Numeric]: Validator[T] = inRange[T](implicitly[Numeric[T]].zero, implicitly[Numeric[T]].zero)
+  def positive[T: Numeric]: Validator.Primitive[T] = Min(implicitly[Numeric[T]].zero, exclusive = true)
+  def positiveOrZero[T: Numeric]: Validator[T] = positive[T].or(zero[T])
```

Comment:
I think using `Min(implicitly[Numeric[T]].zero, exclusive = false)` will render better in the docs. Then, we can get rid of the `zero` one as well :)

Reply:
I think you are right, at leat for the error case. I'll fix it this evening 👍

Reply:
Fixed